### PR TITLE
feat: enabled custom context for create credential request

### PR DESCRIPTION
**Pull request by amodkala**
Signed-off-by: amod kala <amod.kala@securekey.com>

### CODE REVIEW

**Comment by troyronda** [comment]
@cla-bot check

File: pkg/restapi/vc/operation/operations_test.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const testCreateCredentialRequest = `{
-"context":"https://www.w3.org/2018/credentials/examples/v1",
+"@context": ["https://www.w3.org/2018/credentials/v1"],
```
Comment (DRK3) [inline pkg/restapi/vc/operation/operations_test.go]: Per our discussion, try adding some unit tests

**Comment by DRK3** [comment]
Squash/combine the two commits together into one and then I'll approve

**Comment by fqutishat** [comment]
You need to update sandbox project after merging this PR https://github.com/trustbloc/edge-sandbox/blob/29973fa542c0f2f8eeeeb2e3075600ac4bebc744/pkg/restapi/issuer/operation/operations.go#L275

**Pull request merged**